operator/controllers/http/httpscaledobject_controller_test.go

```diff
@@ -0,0 +1,73 @@
+package http
+
+import (
+	"github.com/kedacore/http-add-on/operator/controllers/http/config"
+	"github.com/stretchr/testify/require"
+	"testing"
```

Comment:
We split the imports in 3 groups:
- runtime
- deps
- internals
```suggestion
	"testing"
	
	"github.com/stretchr/testify/require"

	"github.com/kedacore/http-add-on/operator/controllers/http/config"
```

(it's one of the static checks' failure)